setup.py

```diff
@@ -69,19 +69,7 @@ def run(self):
     maintainer='James Douglass',
     maintainer_email='jdouglass@stanford.edu',
     url='http://github.com/natcap/invest',
-    namespace_packages=['natcap'],
-    packages=[
-        'natcap',
-        'natcap.invest',
-        'natcap.invest.coastal_blue_carbon',
-        'natcap.invest.delineateit',
-        'natcap.invest.ui',
-        'natcap.invest.ndr',
-        'natcap.invest.sdr',
-        'natcap.invest.recreation',
-        'natcap.invest.scenic_quality',
-        'natcap.invest.seasonal_water_yield',
-    ],
+    packages=find_namespace_packages('src'),
```

Comment:
Great!  It was not immediately clear to me from this function name that normal packages within `natcap.invest` would also be captured, but that does seem to be the behavior.

Reply:
Yeah, it's confusing! https://github.com/pypa/setuptools/discussions/3427